apps/dashboard/src/core-ui/batch-upload/batch-table.tsx

```diff
@@ -174,9 +173,9 @@ export const BatchTable: React.FC<BatchTableProps> = ({
                 {headerGroup.headers.map((column, i) => (
                   // biome-ignore lint/suspicious/noArrayIndexKey: FIXME
                   <Th {...column.getHeaderProps()} border="none" key={i}>
-                    <Text as="label" size="label.sm" color="faded">
+                    <p className="text-muted-foreground">
```

Comment:
since it was `as="label"` before why not use `<label>` here?

Reply:
It's because `A form label must be associated with an input.biomelint/a11y/noLabelWithoutControl`

and I didn't want to add a comment to suppress it. (The look is about the same fwiw)